parsers/lib/quality.py

```diff
@@ -28,9 +36,7 @@ def validate_exchange(item, k):
     if data_time > arrow.now():
```

Comment:
Maybe validate_year should also validate points in the future?

Reply:
good point